### ISSUE TITLE
[EN][Raid-Cata] Blackwing Descent

### ISSUE THREAD

**Issue by Varchord** [Atlantiss/BugTracker#3143]

**Comment by Varchord** [comment]
I've taken liberty to merge, update and translate all of my existing issues regarding BWD into one, big issue. I hope it will be easier to read and managed by devs. Also, I couldn't sleep :P

**Comment by Isuel** [comment]
what about not being able to shred magmaw's head during the burst phase? or not being able to charge/shadowstep it :P

**Comment by Kejmil** [comment]
There is small area on the veeeery left in which you can backstab/shred him, http://bossblueprint.com/wow/view/:JbRloVSD yellow or blue i think (on the very edge), just watch out or you will jump into lava :P
The source of the problem is that the head isn't really on the spike, only visually - physically it is still up there, on magmaw corpse (it is also causing problems with autoatacks because of this)

**Comment by Varchord** [comment]
It wasn't in original reports, that's why it's not here. Besides, shadowstep has it's own report(two actually which is weird) https://github.com/Atlantiss/BugTracker/issues/2283 https://github.com/Atlantiss/BugTracker/issues/1868

You can shred/backstab during head phase. I see that Kejmil already commented on this. Right now I have no idea if it's intended to work like this or not. If you can provide me with some videos where I can see shredding/backstabing in other places than [this](http://bossblueprint.com/wow/view/:JbRloVSD) then I will add this to original post

**Comment by nerv234** [comment]
Confirm, Magmaw on exposed phase is'nt backstabable.

**Comment by Varchord** [comment]
Show me it should be

**Comment by fafal** [comment]
Can u add this one? https://github.com/Atlantiss/BugTracker/issues/2602

**Comment by Jenovas** [comment]
Magmaw shouldn't switch to phase two while being impaled
https://youtu.be/YUFeRlkwvrQ?t=432
http://kets-emptyslot.blogspot.com/2011/02/magmaw-hc.html
Should be fixed in rev 4430

**Comment by sunshineinabox** [comment]
For the Omnotron reset problem. It only resets when you exit the doorway to the room from my experience. So no one in the raid must leave the room. The Nefarian bug for lava is still preset attempted it today. the whole encounter seems a little buggy and the timing doesn't seem right on Onyxia's abilities either. It might be possible that my DBM was bugged tho so that may not be accurate.

**Comment by Varchord** [comment]
You don't have very much experience then :P

**Comment by Nukeey** [comment]
We found a pretty good way to not have ODS respawn on us. I think it only happens if you deal the killing blow to the "old" boss. To counteract this we just wait for a new bot to get active and focus on that one.

**Comment by Isuel** [comment]
yeah, also noticed it. i was tanking and i got the killing blow on the boss which wasn't the one who was supposed to be attacked, making them reset.

**Comment by sunshineinabox** [comment]
So I have seen (this)[https://github.com/Ksenon/ArkCORE-MoP/blob/master/src/server/scripts/EasternKingdoms/BlackwingDescent/boss_bwd_nefarian.cpp] and it seems he allows flying to get around it not sure if Lava actually rises using this haven't compiled the core to test it.

**Comment by Varchord** [comment]
for the love of god

IT'S NOT THE LAVA THAT IS RISING, IT'S PLATFORM THAT GOES DOWN.

It's being worked on. Stop spamming about it every few days

**Comment by Wolffenstein** [comment]
Check the platform again in rev 4522.

**Comment by Varchord** [comment]
O.o he is alive

**Comment by Skeetis** [comment]
still bugged ty for fix

**Comment by Malylolek** [comment]
Normal and Heroic confirm - still dont work ; (

**Comment by Varchord** [comment]
That was... expected

**Comment by Wolffenstein** [comment]
Rev 4524

**Comment by Kejmil** [comment]
nefarian & onyxia
http://www.wowhead.com/spell=77826
has 180 degree, should have 60

**Comment by xolon** [comment]
on Maloriak, there should be 11 "Cells" with aberrations on left and right wall from which should adds 
"break out" when maloriak casts release aberration, now they just appear from nowhere either on left or right side and there are no "Cells"

**Comment by Frainzil** [comment]
A noted bug for nefarian:
In the 3rd phase, everyone knows that nefarian 'resurrects' the bone construct by throwing shadowblaze spark on 1 of the construct's position. Now The fire that spawns when the spark reaches the floor, does NOT move. It will only spread when the OT, tanking the bone construct, let the adds get hit by fire and than starts to backpaddle. 
https://www.youtube.com/watch?v=HfN3NCHbihY
Watching this video from a retail guild, you can clearly notice that fire actually needs to 'chase' players inside the raid instance, and not just 'stand still'. 
With this not working encounter becomes a bit easier since this means boss actually never needs to move since there is no fire coming close at any time (if tanked well) AND other dead construct will not get reanimated by expanding fire, since it doesn't move. 

Now i also had a question to others; how does the shadowblaze spark works? When we were doing a 10m normal kill recently, i noticed some of the bone construct NEVER got reanimated, since nefarian was always targetting living bone construct to throw his blaze on. Any idea how this was on retail? Is he supposed to target a random construct, or will nef try to reanimate always dead constructs first?

**Comment by sunshineinabox** [comment]
Nefarrian is once again very difficult. The adds are spawned below the platform so in phase 3 when he does his flame its invisible and will wipe the raid.

**Comment by fafal** [comment]
Maloriak - some (A) guild using Mage/Rogue/Night elf outside the room in fight, to leaving from combat and using Mass Resurection.
Not always work but sometimes yes.
Please check and Fix it.

**Comment by Kejmil** [comment]
doesn't we have some "pulse aggro mechanic" on the bosses with idiotically high range, meaning that no one is able to finish the cast of mass ress? or maybe this mechanic is respecting los and this is causing problem here?

**Comment by Varchord** [comment]
Is it me or are there check boxes next to listed bugs in my post? :P

**Comment by Quban** [comment]
> When bosses are at low hp or dying there is a chance that their HP will reset back to 100%. Encounter continues as if nothing have happened. Berserk timer or any other for that matter don't reset.
> Apparently this is is happening pretty often

This bug is probably caused by getting killing blow on the tron that has lower energy

**Comment by PierwszyKrytykRP** [comment]
UPDATE rev 4557 [Dominion](http://www.wowhead.com/spell=79318/dominion)

**Comment by Aival94** [comment]
I am not totally sure but seem tha [Chimaeron](http://www.wowhead.com/npc=43296/chimaeron) [Massacre](http://www.wowhead.com/npc=43296/chimaeron) ability is a little bugged as it also deal 2 times damage(one is melee and other is the normal huge damage you get for it).

For example i was the healer as 17k health and he use massacre and i died.When i check Combat log i saw that i get a melee damage of 7842(or sth like this,forgot yo take screenshot) and then the overkill of 395k(which is normal i as i was after the damage less than 10k)

**Comment by PierwszyKrytykRP** [comment]
DELETED Nefarian

**Comment by Aival94** [comment]
If you ress(either after a wipe or during the encounter using combat ress and soulstone) in Nefarian platform you fall from the ground,die and teleport out of the raid instance(even if your raid kill nefarian you lose the achievement and the raid id of killing nefarian).

Note:BTW did you check Massacre ability if bugged as i said or not?(cannot find any melee swing attack of it anywhere being told)

**Comment by eryczek1615** [comment]
On Maloriak HC should be Nefarian.

**Comment by aneq** [comment]
today we found out that if you kite nefarian's adds far away enough from him on the platform he stops casting shadowblaze spark

**Comment by xolon** [comment]
sometimes magmaws head is in his belly

**Comment by xolon** [comment]
@aneq similar spells which have to land on target can be avoided by the distance, etc on ascendant council if you stand far enough bombs will just fall underground instead hitting the ground where you stand

**Comment by PierwszyKrytykRP** [comment]
UPDATE 13.11.2015
 * Added Nefarian


@xolon Dafq does 
> "magmaws head is in his belly" 

means?

**Comment by xolon** [comment]
sometimes you can see his head inside his stomach, its visual bug i think

**Comment by namreeb** [comment]
Since this hasn't gotten any feedback for a while, be aware that there is still some problem with the Onyxia/Nefarian platform height.  Specifically, pet pathing has issues and combat resses will put the resurrected player under the platform (falling to their death).

**Comment by Samineh** [comment]
Bump :+1:
